### ISSUE TITLE
Cant find much information about the Aurelia Router

### ISSUE THREAD

**Issue by evanlarsen** [aurelia/documentation#362]
The aurelia.io website has information on how to configure a router but is lacking information on:

- router page lifecycles: canDeactivate, canActivate, deactivate, activate
- how to use the router like using the route-href custom attribute or how to actually navigate to different routes.
- the router events which are published by the eventaggregator: 
   - router:navigation:processing
   - router:navigation:error
   - router:navigation:canceled
   - router:navigation:success
   - router:navigation:complete

**Comment by StrahilKazlachev** [comment]
For the lifecycle - [The Route Screen Activation Lifecycle](https://aurelia.io/docs/fundamentals/cheat-sheet#routing).

**Comment by evanlarsen** [comment]
@StrahilKazlachev Thank you, I must have missed that.

**Comment by StrahilKazlachev** [comment]
Also a bit down there is short(really *short*) info on `route-href` and `router.navigateToRoute`/`router.generate`.

**Comment by evanlarsen** [comment]
@StrahilKazlachev  Oh wow!, I didn't realize all the router stuff was in the cheat sheet. Very helpful thanks! I've been scouring the internet for information.

**Comment by StrahilKazlachev** [comment]
It probably never got migrated/added to the dedicated page, the *cheat-sheet* was the only documentation in the beginning(`1.0.0-beta.x`).
Contributions are welcomed, as always.

**Comment by bigopon** [comment]
@evanlarsen It would be nice if you could also help with some missing pieces in the cheat sheet: `from-view`, `capture` binding command and their underlying `from-view` binding mode, capture event listener. They are in the normal doc 😄

**Comment by evanlarsen** [comment]
I'm confused on how to contribute to the documents? I've looked over this repository and it seems that the documents are not stored here.

**Comment by bigopon** [comment]
It's here https://github.com/aurelia/framework/blob/master/doc/article/en-US/cheat-sheet.md 😓

**Comment by Alexander-Taran** [comment]
@evanlarsen you can do it!!!

**Comment by EisenbergEffect** [comment]
@evanlarsen We'd love to have you contribute. In an effort to make it easier, you can not find all documentation in this repo under current/en-us. The folder structure matches the ToC on the site itself.

**Comment by evanlarsen** [comment]
Thank you for making the documentation much easier to contribute to. I will try to do my best to add a contribution when I have time but I'm a bit timid because I've only used Aurelia on a few small projects and I have not looked at the internals. I've only followed documentation on the internet. So, my knowledge would just be pulling information from other sources.

Feel free to close this issue if you want.  Thanks
